.github/workflows/publish-alpha.yml

```
@@ -0,0 +1,32 @@
+name: "publish to npm"
```

Maybe this to make it clear which npm command is being run:
```suggestion
name: "publish alpha release to npm"
```

---

Nice catch!